test/Microsoft.Sbom.Targets.Tests/AbstractGenerateSBomTaskInputTests.cs

```diff
@@ -78,8 +83,11 @@ public void Startup()
             PackageName = packageName,
             PackageVersion = packageVersion,
             NamespaceBaseUri = namespaceBaseUri,
-            ManifestInfo = this.SbomSpecification.ToString(),
-            BuildEngine = this.buildEngine.Object
+            ManifestInfo = this.SbomSpecification,
+            BuildEngine = this.buildEngine.Object,
+#if NET472
```

Comment:
Just for my learning, when we run these tests for net472, it will pull the net472 version of GenerateSbom correct? And then it will set the SbomToolPath to the CLI tool path and invoke the ToolTask version?

Reply:
Yes. That's happening due to what you did with the partial classes. Plus I added macros for only using the SbomTooLPath for the Net 472 tests.